input/pagecontent/changes.md

```diff
@@ -1,6 +1,17 @@
+###  Release 1.0.0
+- Expected Publication date: 2025-01
+- Expected Publication status: Normative
```

Comment:
Incorrect. Publication status matches the status of the spec. AU Core R1 will not be normative.

Reply:
Publication status changed to "TBC".

Reply:
Confirmed with @brettesler-ext and @dbojicic 'Working Standard'.